tutorials/spatiotemporal-image-statistics/index.md

```diff
@@ -0,0 +1,218 @@
+  var endRangeL = startRangeL.advance(interval, intervalUnit);
+  var temporalStat = dataset.filterDate(startRangeL, endRangeL)
+                              .reduce(temporalReducer);
+  
+  // Calculate zonal statistics.
+  var statsL = temporalStat.reduceRegions({
+    collection: regions,
+    reducer: spatialReducers,
+    scale: dataset.first().projection().nominalScale(),
+    crs: dataset.first().projection()
+  });
+  
+  // Set start date as a feature property.
+  return statsL.map(function(feature) {
+    return feature.set({
+      composite_start: startRangeL.format('YYYY'),  // or 'YYYY-MM-dd'
+    });
+  });
+});
+
+zonalStatsL = ee.FeatureCollection(zonalStatsL).flatten();
+
+print('Spatiotemporal statistics (long)', zonalStatsL);
+
+Export.table.toDrive({
+  collection: zonalStatsL,
+  description:'zonal_stats_long',
+});
+```
+
+### Wide table
+
+The wide table will have one row for each region (35 rows in this case) with
+a column per unique combination of statistic and time window. The new column
+names are the concatenation of the statistic and the time window separated
+by an underscore. The process uses a client-side for loop on the number
+of time windows, and for each one, zonal statistics are calculated and appended
+as new properties to the FeatureCollection. An alternative approach is to use
+Earth Engine's `iterate` function, but a comparison of the approaches'
+performance was equal, so we've choosen the for loop for improved readability.
+
+```js
+var temporalStatW;  // defined dynamically for each temporal image composite
+var startRangeW;  // defined dynamically for each temporal image composite
+var reduce = function(feature) {
+  // Calculate zonal statistics.
+  var statsW = temporalStatW.reduceRegion({
+    reducer: spatialReducers,
+    geometry: feature.geometry(),
+    scale: dataset.first().projection().nominalScale(),
+    crs: dataset.first().projection()
+  });
+  
+  // Append date to the statistic label.
+  var keys = ee.Dictionary(statsW).keys();
+  var newKeys = keys.map(function(key) {
+    return ee.String(key).cat('_')
+               .cat(startRangeW.format('YYYY'));  // or 'YYYY-MM-dd'
+  });
+  
+  // Add the statistic properties to the feature.
+  return feature.set(statsW.rename(keys, newKeys));
+};
+
+var zonalStatsW = regions;  // make a copy of the regions FeatureCollection
+
+// Loop through sequence of intervals to calculate statistics for each.
+for (var i = 0; i < intervalCount; i++) {
+  var startRangeW = ee.Date(startDate).advance(i, intervalUnit);
+  var endRangeW = startRangeW.advance(interval, intervalUnit);
+  temporalStatW = dataset.filterDate(startRangeW, endRangeW).mean()
+                              .reduce(temporalReducer);
+  zonalStatsW = zonalStatsW.map(reduce);
+}
+
+print('Spatiotemporal statistics (wide)', zonalStatsW);
+
+Export.table.toDrive({
+  collection: zonalStatsW,
+  description:'zonal_stats_wide',
+});
+```
+
+## Acknowledgements
```

Comment:
This section isn't necessary since the commit history is public on GitHub, and since future authors not mentioned here may make tweaks or adjustments. Is there a reason to include an additional attribution here?

Reply:
Good points, Gino. I'll remove the section.